File: Python Book/4. Simple Conditions/13_area_of_figures/area_of_figures.py
import math

figure = input().lower()
Square = "square"
Rectangle = "rectangle"
Circle = "circle"
Triangle = "triangle"

if figure == Square:
    a = float(input())
    print(round(a*a, 3))
elif figure ==  Rectangle:
    a = float(input())
    b = float(input())
    print(round(a*b, 3))
elif figure == Circle:
    r = float(input())
    pi = math.pi
    print(round(pi * r * r, 3))
elif figure == Triangle:
    a = float(input())
    h = float(input())
    print(round((a*h)/2, 3))

File: JavaScript/Programming Basics - JavaScript/Conditional Statements/areaOfFigures.js
function getArea(input) {
    let figure = input[0];
    let sideA = Number(input[1]);
    let area = undefined;
    if (figure === 'square') {
        area = sideA * sideA;
    } else if(figure === 'rectangle') {
        let sideB = Number(input[2]);
        area = sideA * sideB;
    } else if (figure === 'circle') {
        area = Math.PI  * Math.pow(sideA, 2);
    } else if (figure === 'triangle') {
        let lenght = Number(input[2]);
        area = (sideA * lenght) / 2;
    }
    console.log(area.toFixed(3));
}

getArea(["square", "5"]);
getArea(["rectangle", "7", "2.5"]);
getArea(["circle", "6"]);
getArea(["triangle", "4.5", "20"]);